terraform/deployments/tfc-configuration/govuk-publishing-infrastructure.tf

```diff
@@ -0,0 +1,94 @@
+module "govuk-publishing-infrastructure-integration" {
+  source  = "alexbasista/workspacer/tfe"
+  version = "0.9.0"
+
+  organization      = var.organization
+  workspace_name    = "govuk-publishing-infrastructure-integration"
+  workspace_desc    = "This module manages AWS resources which are specific to GOV.UK Publishing."
+  workspace_tags    = ["integration", "govuk-publishing-infrastructure", "eks", "aws"]
+  terraform_version = "1.7.0"
+  execution_mode    = "remote"
+  working_directory = "/terraform/deployments/govuk-publishing-infrastructure/"
+  trigger_patterns  = ["/terraform/deployments/govuk-publishing-infrastructure/**/*"]
+
+  project_name = "govuk-infrastructure"
+  vcs_repo = {
+    identifier     = "alphagov/govuk-infrastructure"
+    branch         = "main"
+    oauth_token_id = data.tfe_oauth_client.github.oauth_token_id
+  }
+
+  team_access = {
+    "GOV.UK Non-Production" = "write"
+    "GOV.UK Production"     = "write"
+  }
+
+  variable_set_names = [
+    "aws-credentials-integration",
+    "common",
+    "common-integration"
+  ]
+}
+
+module "govuk-publishing-infrastructure-staging" {
+  source  = "alexbasista/workspacer/tfe"
+  version = "0.9.0"
+
+  organization      = var.organization
+  workspace_name    = "govuk-publishing-infrastructure-staging"
+  workspace_desc    = "This module manages AWS resources which are specific to GOV.UK Publishing."
+  workspace_tags    = ["staging", "govuk-publishing-infrastructure", "eks", "aws"]
+  terraform_version = "1.7.0"
+  execution_mode    = "remote"
+  working_directory = "/terraform/deployments/govuk-publishing-infrastructure/"
+  trigger_patterns  = ["/terraform/deployments/govuk-publishing-infrastructure/**/*"]
+
+  project_name = "govuk-infrastructure"
+  vcs_repo = {
+    identifier     = "alphagov/govuk-infrastructure"
+    branch         = "main"
+    oauth_token_id = data.tfe_oauth_client.github.oauth_token_id
+  }
+
+  team_access = {
+    "GOV.UK Non-Production" = "write"
```

Comment:
I think we usually treat staging the same as prod from an access controls perspective. Let's go with just the production group here for now and see if it leads to any friction.
```suggestion
```

(apologies if I'd not spotted this on previous PRs!)

Reply:
Ah good spot, I'll update the rest of the TF projects too